butterfree/_cli/migrate.py

```diff
@@ -88,6 +89,28 @@ def __fs_objects(path: str) -> Set[FeatureSetPipeline]:
 )
 
 
+class Migrate:
+    """Execute migration operations in a Database based on pipeline Writer.
+
+    Attributes:
+        pipelines: list of Feature Set Pipelines to use to migration.
+    """
+
+    def __init__(self, pipelines: Set[FeatureSetPipeline]) -> None:
+        self.pipelines = pipelines
+
+    def _send_logs_to_s3(self) -> None:
+        """Send all migration logs to S3."""
+        pass
+
+    def run(self) -> None:
```

Comment:
We will not parse the feature sets within Butterfree anymore, right?

Reply:
I think this is not necessary anymore :smiley_cat: